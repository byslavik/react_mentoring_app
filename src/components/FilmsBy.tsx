import * as React from 'react';


interface FilmsByProps {
  field: string;
}

export class FilmsBy extends React.Component<FilmsByProps, any> {
    render() {      
      return (<span>Films by {this.props.field}</span>)
    }
}
