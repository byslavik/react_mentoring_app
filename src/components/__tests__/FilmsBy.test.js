import * as React from 'react';
import {FilmsBy} from '../FilmsBy';
import * as renderer from 'react-test-renderer';


describe('FilmsBy component', ()=>{
  it('FilmsBy component shows correctly', () => {
    const tree = renderer.create(
      <FilmsBy field="person"/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})