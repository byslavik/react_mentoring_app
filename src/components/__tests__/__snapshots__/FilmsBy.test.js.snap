// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsBy component FilmsBy component shows correctly 1`] = `
<span>
  Films by 
  person
</span>
`;
